### PR TITLE
1.1 nav to signup page

### PR DESCRIPTION
**Pull request by nazhudha**
Added Signup and Sign-in buttons to navigate between pages, with Cypress tests

### CODE REVIEW

**Review by davekempsell — APPROVED** [review]
looks good, not sure what the .DS_Store file is that's causing a conflict? Haven't seen that before

File: views/home/index.hbs [hunk 1]
```diff
@@ -1,2 +1,14 @@
+<form action="./users/new" method="GET">
+  <input type="submit" value="Sign Up" />
```
Comment (raphaella-rose) [inline views/home/index.hbs]: This could be a link <a> instead for simpler code as it's a get request.

File: views/home/index.hbs [hunk 2]
```diff
@@ -1,2 +1,14 @@
...
+</form>
+
+{{!-- Sign up button implemented --}}
+
+<form action="./sessions/new" method="GET">
```
Comment (raphaella-rose) [inline views/home/index.hbs]: This could be a link instead for simpler code as it's a get request.

File: views/layout.hbs [hunk 3]
```diff
@@ -5,9 +5,9 @@
     <link rel='stylesheet' href='/stylesheets/style.css' />
   </head>
   <body>
-    <form action='/sessions?_method=DELETE' method='POST'>
+    {{!-- <form action='/sessions?_method=DELETE' method='POST'>
```
Comment (raphaella-rose) [inline views/layout.hbs]: Is this commented out? Why?

File: views/sessions/new.hbs [hunk 4]
```diff
@@ -1,3 +1,9 @@
+<form action="/" method="GET">
```
Comment (raphaella-rose) [inline views/sessions/new.hbs]: This could be a link instead for simpler code as it's a get request.

File: views/users/new.hbs [hunk 5]
```diff
@@ -1,3 +1,7 @@
+<form action="/" method="GET">
```
Comment (raphaella-rose) [inline views/users/new.hbs]: This could be a link instead for simpler code as it's a get request.

**Review by raphaella-rose — CHANGES_REQUESTED** [review]
Hey guys, looks good but there are a couple of changes I think need to be made. I'm around for a bit to talk through the comments! :)

**Comment by nazhudha** [comment]
.DS_Store conflict, resubmit pull request

**Pull request closed without merging**